simple_api/django_object/converter.py

```diff
@@ -54,25 +56,29 @@ def convert_django_field(field, field_name, both_fields, input_fields, output_fi
 @convert_django_field.register(DateTimeField)
 @convert_django_field.register(EmailField)
 @convert_django_field.register(PositiveSmallIntegerField)
-def convert_to_primitive_type(field, field_name, both_fields, input_fields, output_fields, field_validators):
+def convert_to_primitive_type(field, field_name, both_fields, input_fields, output_fields):
     assert field.__class__ in DJANGO_SIMPLE_API_MAP, "Cannot convert `{}`".format(field.__class__)
     both_fields[field_name] = DJANGO_SIMPLE_API_MAP[field.__class__](nullable=field.null,
-                                                                     default=get_default(field), exclude_filters=())
+                                                                     default=get_default(field), exclude_filters=(),
+                                                                     validators=[DjangoValidator(field_name, x)
+                                                                                 for x in field.validators])
 
 
 @convert_django_field.register(ForeignKey)
 @convert_django_field.register(OneToOneField)
-def convert_to_object_type(field, field_name, both_fields, input_fields, output_fields, field_validators):
+def convert_to_object_type(field, field_name, both_fields, input_fields, output_fields):
     target_model = field.remote_field.model
     target_pk_field_name, target_pk_field = get_pk_field(target_model)
     converted_pk_field = DJANGO_SIMPLE_API_MAP[target_pk_field.__class__]
-
-    input_fields[field_name + "_id"] = converted_pk_field(nullable=field.null, exclude_filters=())
-    output_fields[field_name] = ObjectType(target_model, nullable=field.null, exclude_filters=())
+    validators = [DjangoValidator(field_name, x) for x in field.validators]
+    validators.append(ForeignKeyValidator(target_model))
+    input_fields[field_name + "_id"] = converted_pk_field(nullable=field.null, exclude_filters=(),
+                                                          validators=validators)
+    output_fields[field_name] = ObjectType(target_model, nullable=field.null, exclude_filters=(), validators=validators)
```

Comment:
output field potrebuje validatory? asi to nevadi, ale je to zbytecne a pri cteni kodu muze byt matouci

Reply:
Nepotřebuje, byla to jedna věc co jsem přehlédl že jsem pokazil když jsem se zbavoval commitu tak se změnila...